### DISCUSSION TITLE
Chart legend - Size

### DISCUSSION THREAD

**Discussion by thomai-d** [datalust/seq-tickets#1101] [Ideas]
It would be great if the chart legends would use the provided space more efficiently.

In the following example, the legend could use more space to display the full label - either automatically or by manually extending it:

![grafik](https://user-images.githubusercontent.com/24974298/58502734-6f563700-8187-11e9-89fa-c10294f7d889.png)

**Comment by nblumhardt** [comment]
Thanks for the note, @thomai-d, agreed we should improve this.
